src/controller/java/AndroidCallbacks.cpp

```diff
@@ -253,15 +253,11 @@ void ReportCallback::OnAttributeData(const app::ConcreteDataAttributePath & aPat
 
     jobject value = DecodeAttributeValue(aPath, readerForJavaObject, &err);
     // If we don't know this attribute, suppress it.
-    if (err == CHIP_ERROR_IM_MALFORMED_ATTRIBUTE_PATH_IB)
+    if (err != CHIP_NO_ERROR)
```

Comment:
Comment is probably wrong now and should be updated.
Maybe just say `err = CHIP_NO_ERROR` then without the if.

Reply:
is it guaranteed that value is null (or some other valid java type) if decode fails?